CHANGELOG.md

```diff
@@ -12,8 +12,6 @@ It also includes a few other bug fixes and performance improvements.
 
```

Comment:
No need to touch this flie. It is autogenerated (it will update in the next point release)

Reply:
Ok, I'll revert the changes